dev/VSIX/Extension/source.extension.vsixmanifest

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
   <Metadata>
-    <Identity Id="Microsoft.WindowsAppSDK" Version="|%CurrentProject%;GetVSIXVersion|" Language="en-US" Publisher="Microsoft" />
+    <Identity Id="Microsoft.ProjectReunion" Version="|%CurrentProject%;GetVSIXVersion|" Language="en-US" Publisher="Microsoft" />
```

Comment:
Should the identity differ between experimental and stable/preview builds, so as not to collide?

Reply:
It already does. See the XmlPoke stuff in the csproj. It detects if `EnableExperimentalVSIXFeatures` is set, and dynamically updates the `Identity`, `DisplayName`, `Tags`, and (new to this PR) `Preview` fields in the manifest.

Reply:
The Identity is modified by the XmlPoke task on line 306 of WindowsAppSDK.Extension.csproj